tests/system/robot/NVDAInstaller.robot

```diff
@@ -15,29 +15,23 @@ Test Setup	default startup
 Test Teardown	default teardown
 
 *** Keywords ***
-default exit if no installDir
-	Pass Execution If	not r'${installDir}'	"skipping due to not running on installer"
```

Comment:
I'd suggest a suite setup (not test setup) step that fails if this is missing. Failing should fail all the tests.